packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts

```diff
@@ -184,7 +184,7 @@ export class HostedPluginManagerClient {
         if (this.pluginLocation) {
             const fsPath = await this.fileService.fsPath(this.pluginLocation);
             if (fsPath) {
-                outFiles = [new Path(fsPath).join('**', '*.js').toString()];
+                outFiles = [new Path(fsPath).join('out', '*.js').toString()];
```

Comment:
What about extensions that put their files in `lib`? What about extensions not using TypeScript that put their .js files at the root, like just `index.js`?

Reply:
We are talking about compiled extensions and AFAIK most extensions configure their compiled code in the `out` directory. 
We could add additional most common folders to `outFiles` - like `dist` or `lib`. 

Without it we will get the problem we have here - too many js files > load timeout > extension is not debuggable. 

What do you think?

Reply:
I really don't like hardcoding configurations like this. What about adding a preference to control the way `outFiles` is defined? Make it so someone can write `${pluginPath}` in the strings to be replaced by `new Path(fsPath)`, and have the default be `outFile = undefined`?

Reply:
I added ability to configure `outFiles` as you outlined. Please review again.
Thanks

Reply:
@marechal-p bump